contracts/src/v0.8/workflow/dev/v2/WorkflowRegistry.sol

```diff
@@ -36,21 +36,23 @@ contract WorkflowRegistry is Ownable2StepMsgSender, ITypeAndVersion {
   /// TAG_SHIFT  (8 bits)   = 8
   /// URL_SHIFT  (8 bits)   = 16
   /// ATTR_SHIFT (16 bits)  = 24
+  /// EXPIRY_SHIFT (32 bits) = 40
   ///
   /// @dev Single-slot override; when s_metadataCaps == 0, all defaults apply.
   /// @dev DEFAULT_PACKED bundles the defaults:
   ///   (uint256(64)  << NAME_SHIFT) │
   ///   (uint256(32)  << TAG_SHIFT)  │
   ///   (uint256(200) << URL_SHIFT)  │
   ///   (uint256(1024)<< ATTR_SHIFT)
-  /// where maxNameLen = 64, maxTagLen = 32, maxUrlLen = 200, maxAttrLen = 1024
+  /// where maxNameLen = 64, maxTagLen = 32, maxUrlLen = 200, maxAttrLen = 1024, maxExpiry = 604800
```

Comment:
The packing doesn't make sense anymore if this is uint256 as that's just a slot on its own. Let's discuss and I think adding this is the right place to moving the config out of the compile time const into a struct